engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/moduleDetailsScreen/ModuleDetailsScreen.java

```diff
@@ -192,9 +193,7 @@ public String getString(Module value) {
     }
 
     private List<DependencyInfo> getSortedDependencies(final Module module) {
-        return module.getMetadata()
-                .getDependencies().stream()
-                .sorted(Comparator.comparing(DependencyInfo::getId))
+        return module.getMetadata().getDependencies().stream().sorted(Comparator.comparing(DependencyInfo::getId))
```

Comment:
It seems like this file is just formatting changes. Some of them are good but also some of them i'd argue actually make it less clear. In particular the ones involving the chained calls like this.

We should probably just discard all the changes to this file as it seems superfluous. Also seems like formatting issues are the biggest problem here haha

Reply:
Entirely agree. All changes discarded.